portfolio/src/main/webapp/style.css

```diff
@@ -34,10 +38,13 @@ body {
 }
 
 #sidebar a {
-  color: #988;
   padding: 10px 15px;
 }
```

Comment:
Not sure if this would be configured here, but I noticed that sometimes due to the close spacing of contents of the sidebar, I accidentally click on one of the neighboring pages instead.

Reply:
I had a bit of a hard time with this, but I managed to increase the spacing by changing the vertical alignment of the links to "middle"... Also, I made the text bigger, so hopefully that will help too.

Reply:
Cool, yes this does help!